src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -80,6 +85,12 @@ public enum Notification {
     /** If this user was created by an {@link ApiUser}, this parameter will match the {@link ApiUser}'s id */
     public String applicationId;
 
+    /** Companions and observers of this user. */
+    public List<RelatedUser> relatedUsers = new ArrayList<>();
```

Comment:
We need to update the `preCreateHook` and `preUpdateHook` so that if the list of related users (and dependents too?) changes, the new users are sent an email with a link to the new `/acceptdependent` endpoint you have added.

Reply:
WIP: https://github.com/ibi-group/otp-middleware/pull/256/commits/bacb87e3ce664a39ffc25e8e7101e08cab00553e

Need a few questions answering and consider testing.

Reply:
Do we need the dependent's name as well, to include in the email going to related users?